Sources/SecureXPC/Client/XPCClient.swift

```diff
@@ -126,9 +126,10 @@ public class XPCClient {
         XPCMachClient(serviceName: machServiceName)
     }
 
-	// MARK: Implementation
+    // MARK: Implementation
 
     internal let serviceName: String
```

Comment:
Should this variable still exist in the super class? While both subclasses currently do make use of it, seems a bit off from an encapsulation point of view as this class itself does not. (I realize this isn't new behavior as of this PR, but just noticed it now.)

Reply:
Sure, could you open a new issue about refactoring this?

Reply:
Actually looking at #23 I think this should be addressed, see that PR for relevant comments.